vulnerability/runner.go

```diff
@@ -51,8 +51,9 @@ func NewVulnerabilityRunner(log *logp.Logger) (VulnerabilityRunner, error) {
 			RekorURL: "https://rekor.sigstore.dev",
 		},
 		DBOptions: flag.DBOptions{
-			NoProgress:   true,
-			DBRepository: "ghcr.io/aquasecurity/trivy-db",
+			NoProgress:       true,
+			DBRepository:     "ghcr.io/aquasecurity/trivy-db",
+			JavaDBRepository: "ghcr.io/aquasecurity/trivy-java-db",
```

Comment:
before adding `JavaDBRepository` this used to error with:
```
Java DB update error: DB download error: repository name error (:1): could not parse reference: :1
```

probably due to a missing value. now it errors with:
```
Java DB open error: can't open db: sql: unknown driver "sqlite"
```

Reply:
^ this one is fixed by including the sqlite package